config/environments/production.rb

```diff
@@ -58,14 +58,29 @@
   # config.cache_store = :mem_cache_store
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = 'http://assets.example.com'
+  # config.action_controller.asset_host = "http://assets.example.com"
 
   # Ignore bad email addresses and do not raise email delivery errors.
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
+  config.active_record.dump_schema_after_migration = false
+  config.action_mailer.raise_delivery_errors = true
+  host = "pure-beyond-21572.herokuapp.com"
+  config.action_mailer.default_url_options = { host: host }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :address        => "smtp.gmail.com",
```

Comment:
1. `:address        =>`
sửa thành `address:`

2. tìm hiểu gem figaro in rails để setting các giá trị vào biến môi trường